pyInterface/scripts/printMetadata.py

```diff
@@ -17,7 +16,6 @@
 
 	parser = argparse.ArgumentParser(description="print event metadata")
 	parser.add_argument("inputFile", type=str, metavar="inputFile", help="input file in ROOTPWA format")
-	parser.add_argument("-a", "--amplitudeObjectBaseName", type=str, metavar="objectBaseName", help="amplitude object base name (default: try to guess from input file name)")
 	parser.add_argument("-r", action="store_true", dest="recalculateHash", help="recalculate hash and compare it with the stored one (default: %(default)s)")
```

Comment:
Unrelated to your changes: wouldn't
`-v`, `--verify`, `verify hash(es)`
be a better naming for this option?
However, I'm not sure, whether it would be worth breaking the interface.